node/node.go

```diff
@@ -199,6 +199,12 @@ func GetCommand() *cobra.Command {
 
 			// This blocks until the context is finished or until an error is produced
 			err = app.Start(ctx)
+			if err != nil {
+				app.log.Error("app failed", log.Err(err))
+			} else {
```

Comment:
Without `With()` it will be sugared and display weird things instead of the text:

```suggestion
			if err != nil {
				app.log.With().Error("app failed", log.Err(err))
			} else {
```